tests/platform_tests/cli/test_show_chassis_module.py

```diff
@@ -0,0 +1,82 @@
+import logging
+import pytest
+from tests.common.helpers.assertions import pytest_assert
+from util import get_field_range, get_fields, get_skip_mod_list
+
+
+pytestmark = [
+    pytest.mark.topology('t2')
+]
+
+CMD_SHOW_CHASSIS_MODULE = "show chassis-module"
+
+
+def parse_chassis_module(output, expected_headers):
```

Comment:
If the output is of tabulate format, you can consider using this method to parse the output: https://github.com/Azure/sonic-mgmt/blob/master/tests/common/devices.py#L1034

Reply:
@wangxin I need to verify headers for show command which is done in parse function which is already using get_field_range, get_fields utils from utils under cli folder